apis/python/node/Cargo.toml

```diff
@@ -25,7 +25,8 @@ arrow = { workspace = true, features = ["pyarrow"] }
 pythonize = { workspace = true }
 futures = "0.3.28"
 dora-ros2-bridge-python = { workspace = true }
-pyo3_special_method_derive = "0.2.1"
+pyo3_special_method_derive = "0.3.0"
+pyo3_special_method_derive_lib = "0.3.0"
```

Comment:
This is because `pyo3_s.m.d` is a proc macro crate, so it cannot export any traits.  `pyo3_s.m.d_lib` exports the `PyDisplay` and `PyDebug` traits as well as some implementations.

Reply:
It seems that people tend to reexport the proc macro crate within the lib crate.

See: https://stackoverflow.com/questions/64288426/is-there-a-way-to-have-a-public-trait-in-a-proc-macro-crate

but we can merge this in a follow up PR.

Reply:
Ah ok, thanks for letting me know.